src/client/Zecko.ts

```diff
@@ -98,7 +98,24 @@ export class Zecko {
    *  {
    *    // `YOUR_PAYMENT_METHOD` can be one of `COD` or `PREPAID`
    *    paymentMethod: 'YOUR_PAYMENT_METHOD',
-   *  }
+   *    discountCode: 'YOUR_DISCOUNT_COUPON_CODE',
+        shippingAddress: {
+          // Typically for the street address or PO Box number
+          address1: 'YOUR_CUSTOMER_ADDRESS',
```

Comment:
Please check indentation here, does not look good in file:///Users/kartikarora/repos/zecko/zecko-js/docs/classes/Zecko.html#constructor